sp/src/game/server/episodic/grenade_hopwire.cpp

```diff
@@ -2688,6 +2711,14 @@ void CGrenadeHopwire::Detonate( void )
 
 	// Get the time until the apex of the hop
 	float apexTime = sqrt( hopHeight / GetCurrentGravity() );
+
+	// If this is a stasis grenade, use the stasis grenade think function to detonate
+	if (FClassnameIs(this, "npc_grenade_stasis"))
```

Comment:
This could use a new virtual function instead of a classname check. Maybe `HopwireStyle` could be moved to `grenade_hopwire.h`. Not a big deal right now though since this only shows up in one place